slyd/slyd/html.py

```diff
@@ -82,13 +74,13 @@ def descriptify(doc, base=None, proxy=None):
             else:
                 for key, val in element.attributes.copy().items():
                     # Empty intrinsic events
-                    if key in INTRINSIC_EVENT_ATTRIBUTES:
+                    if key.startswith('on') or key == "http-equiv":
                         element.attributes[key] = ""
                     elif base and proxy and key == "style" and val is not None:
                         element.attributes[key] = process_css(val, -1, base)
                     # Rewrite javascript URIs
                     elif key in URI_ATTRIBUTES and val is not None:
-                            if "javascript:" in _deentitize_unicode(val):
+                            if "javascript:" in _deentitize_unicode(val).lower():
```

Comment:
Sounds good, I'll add the test cases to the tests and update the PR.
Is protect_xss function necessary? It looks over-protective to me since replaces characters like /, &, ? and % that are normal to find in an url.